public/docs/ts/latest/guide/index.jade

```diff
@@ -120,10 +113,10 @@ block example-links
 
   # Feedback
 
-  We welcome feedback! Leave a comment by clicking the icon in upper right corner of the banner.
+  We welcome feedback! To leave a comment click the icon in the upper right corner of the banner.
 
-  Post *documentation* issues and pull requests on the
+  Post documentation issues and pull requests on the
```

Comment:
"Post" confused me. (I at first read it with the meaning "after documentation".) Also, it's GitHub, not github. Maybe:

Use the [angular.io](...) GitHub repo for **documentation** issues and pull requests.